packages/astro/src/core/errors/errors-data.ts

```diff
@@ -771,7 +771,7 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 		code: 9001,
 		message: (collection: string, entryId: string, error: ZodError) => {
 			return [
-				`${String(collection)} → ${String(entryId)} frontmatter does not match collection schema.`,
+				`**${String(collection)} → ${String(entryId)}** frontmatter is invalid.`,
```

Comment:
Flagging for docs review @sarah11918 👀

Reply:
@bholmesdev I love the change overall! And, I know you wanted something shorter here, but I don't *love* "invalid" vs "does not match what you said you wanted here." (which is *why* it's invalid)

If you feel the need to shorten, something like ...

"unexpected frontmatter" feels a little more user friendly?

Your call! Non-blocking!

Reply:
Understood! Didn't feel too strongly about this change. I think I'll revert to the old wording, with the bolding added 👍